kloppy/infra/serializers/event/wyscout/serializer.py

```diff
@@ -0,0 +1,370 @@
+import json
+import logging
+from typing import Dict, List, Tuple
+
+from kloppy.domain import (
+    BallOutEvent,
+    BodyPart,
+    BodyPartQualifier,
+    CardEvent,
+    CardType,
+    EventDataset,
+    FoulCommittedEvent,
+    GoalkeeperAction,
+    GoalkeeperActionQualifier,
+    Ground,
+    Metadata,
+    Orientation,
+    PassEvent,
+    PassQualifier,
+    PassResult,
+    PassType,
+    Period,
+    PitchDimensions,
+    Player,
+    Point,
+    Provider,
+    Qualifier,
+    RecoveryEvent,
+    SetPieceQualifier,
+    SetPieceType,
+    ShotEvent,
+    ShotResult,
+    TakeOnEvent,
+    TakeOnResult,
+    Team,
+)
+from kloppy.infra.serializers.event import EventDataSerializer
+from kloppy.utils import Readable, performance_logging
+
+logger = logging.getLogger(__name__)
+
+
+def _parse_team(raw_events: List[Dict], wyId: str, ground: Ground) -> Team:
+    team = Team(
+        team_id=wyId,
+        name=raw_events["teams"][wyId]["officialName"],
+        ground=ground,
+    )
+    team.players = [
+        Player(
+            player_id=str(p["playerId"]),
+            team=team,
+            jersey_no=None,
+            first_name=p["player"]["firstName"],
+            last_name=p["player"]["lastName"],
+        )
```

Comment:
Please use meaningful variable name instead of ‘P’

Reply:
Done